CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to the MultiApps CF CLI Plugin
```

Comment:
Shouldn't the below be marked as 1,2,3...

Reply:
The multiapps-controller Contributing doc doesn't have an ordered list either. Nikolay, what do you think? Should I change the lists to numbered?

Reply:
I think it is good the way suggested.